README.md

```diff
@@ -1,26 +1,42 @@
 [![CircleCI](https://circleci.com/gh/giantswarm/awscnfm.svg?style=shield)](https://circleci.com/gh/giantswarm/awscnfm)
 [![GoDoc](https://godoc.org/github.com/giantswarm/awscnfm?status.svg)](https://pkg.go.dev/github.com/giantswarm/awscnfm?tab=overview)
 
+
+
 # awscnfm
 
 Conformance test tool for Kubernetes on AWS by Giant Swarm.
 
+
+
 ## Basic concepts
 
-Tests are organized based on **cluster scopes** containing multiple **actions**. Each cluster scope represents a distinct cluster configuration. The actions take care of things like creating the cluster, checking that it functions as expected, and deleting it.
```

Comment:
Every time somebody writes in markdown files without wrapping their wall of text, a little kitten dies. 😢